src/index.ts

```diff
@@ -1,5 +1,5 @@
-import List from './List.js';
-import { arrayMove, arrayRemove } from './utils.js';
+import List from './List';
```

Comment:
This leads to an error when building a webpack project that uses this module (https://github.com/JetBrains/ring-ui/tree/dependabot/npm_and_yarn/react-movable-3.0.3):

```
ModuleNotFoundError: Module not found: Error: Can't resolve './List' in '/Users/filippriabchun/IdeaProjects/ring-ui/node_modules/react-movable/lib'
Did you mean 'List.js'?
BREAKING CHANGE: The request './List' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

You may want to tweak the eslint config instead

Reply:
`3.0.4` should be fine now